### ISSUE TITLE
Gap analysis - probation accessibility statements

### ISSUE THREAD

**Issue by RoseBrandle72** [ministryofjustice/moj-frontend#2540]
## What

Carry out a gap analysis on the probation accessibility statements to understand:

- which accessibility statements should be published
- if there any currently published that shouldn't be
- if all the teams whose statements we've published are listed in the global nav
- if we should update the paragraph about requesting alternative formats (under 'Feedback and contact information') in all existing statements
- the best contact (email address) to be publishing on the accessibility hub page

## Why

1. we've had advice from Eu about the wording for requesting alternative formats
2. we may be publishing accessibility statements for teams that can't be accessed via the global nav (eg create and vary a licence)
3. we have an old email link on the hub page

## Who needs to be involved

- Content Designer
- Developer
- Product Manager

## Done

When we have  

- confirmed the list of eligible accessibility statements
- updated the accessibility hub page accordingly
- updated the content about requesting alternative formats
- updated the email address on the accessibility hub page

**Comment by RoseBrandle72** [comment]
Advice from Eu:
_If you want to remove the para starting with “If you need information on this website …” still make sure that you have a reliable escalation rout under the heading of “[Feedback and contact information](https://www.gov.uk/guidance/model-accessibility-statement#:~:text=last%20such%20review.%5D-,Feedback%20and%20contact%20information,for%20accessibility%20and%20for%20processing%20requests%20sent%20through%20the%20feedback%20mechanism%5D.,-Enforcement%20procedure)”. Wording is not prescribed._

**Comment by RoseBrandle72** [comment]
I've messaged Eu today with a suggestion for wording for the Feedback and contact information section.

**Comment by RoseBrandle72** [comment]
Audit of current contact wording, to understand how content about requesting alternative formats can be standardised.

https://justiceuk.sharepoint.com/:x:/r/sites/ConnectedServices/Shared%20Documents/Content%20design/PDS%20accessibility%20statements.xlsx?d=wfe0e7737996443e89fe021b5f7c69330&csf=1&web=1&e=LK0CHN

Next steps: contact services with extra wording around alternative formats, to ensure new wording covers their activities. Then we can apply the suggested content updates.

**Comment by RoseBrandleMOJ** [comment]
Response from Eu in accessibility - we don't need the alternative formats wording.

_"You don't need to mention alternative formats, that's not on the mandatory information list."_

**Comment by RoseBrandleMOJ** [comment]
Check date of WMT being shuttered with MaW team, so we can remove accessibility statement.
